apm-protocol/apm-network/src/main/java/org/apache/skywalking/apm/network/trace/component/ComponentsDefine.java

```diff
@@ -130,4 +130,6 @@
 
     public static final OfficialComponent SPRING_WEBFLUX = new OfficialComponent(67, "spring-webflux");
 
+    public static final OfficialComponent PLAY = new OfficialComponent(68, "Play");
```

Comment:
Do you submit Play icon to UI repo? Or do you need one there?

Reply:
Add icon to `skywalking-ui/public/img/node/`? Why all logos is gray?

Reply:
It is required to gray. If your could show good in black background, it is good. Solr icon is red.

Reply:
Check other pull request, and test locally. You could post the screenshot here.

Reply:
https://github.com/apache/skywalking-rocketbot-ui/pull/142

![image](https://user-images.githubusercontent.com/1162070/63910033-afafa480-ca57-11e9-8d37-4a9f8e2b22af.png)